app/views/pages/data_requirements.html.erb

```diff
@@ -0,0 +1,88 @@
+<%= render PageTitle::View.new(title: "pages.data_requirements") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: "/home"
```

Comment:
:thinking: 


![image](https://user-images.githubusercontent.com/470137/104374655-14b74d80-551a-11eb-8dfe-234949467524.png)

Reply:
think @avinhurry is doing the home page at the moment, it's just a placeholder till then